test/utils.ts

```diff
@@ -12,8 +12,7 @@ export const SEED_DATA = [
 ];
 
 export async function seedDatabase() {
-  const database = new PrismaClient();
-
-  await database.$executeRaw`TRUNCATE TABLE "rain_report" CASCADE`;
+  const database = new DatabaseService();
```

Comment:
Small refactor. Use `DatabaseService` and reset primary key on `TRUNCATE`. Remove `CASCADE` as there are no dependent/associated tables.